### PR TITLE
Defining points for testnet categories

### PR DESCRIPTION
**Pull request by leanthebean**
Defining points for promoting the testnet and contributing to the cmmunity categories

## Summary

Screen Shot 2022-02-01 at 10 47 29 AM
(and same for Contributing to the Community section) 

## Testing Plan 
Looked at it on localhost

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[X] No
[ ] Yes
```

### CODE REVIEW

File: components/About/data.tsx [hunk 1]
```diff
@@ -109,7 +109,14 @@ export const callsToAction = {
       title: 'Contributing to the Community',
       content:
         'Help us make Iron Fish more accessible to a wider audience! Published tutorials, articles, forum posts, videos, scripts, and translations are examples of contributions that can earn you points.',
-      comingSoon: true,
+      submissionForm: true,
+      points: [
+        'XSmall = 10 points',
```
Comment (brekk) [inline components/About/data.tsx:114]: Minor, not a blocker, but maybe "Extra Small" instead of "XSmall"?

File: components/About/data.tsx [hunk 2]
```diff
@@ -109,7 +109,14 @@ export const callsToAction = {
...
+        'Small = 100 points',
+        'Medium = 500 points',
+        'Large = 750 points',
+        'XLarge = 1000 points',
```
Comment (brekk) [inline components/About/data.tsx:118]: Samesies

**Review by brekk — APPROVED** [review]
Suggestions but no blockers

**Comment by dguenther** [comment]
I think integration tests need to be updated as well for this.

**Pull request merged**